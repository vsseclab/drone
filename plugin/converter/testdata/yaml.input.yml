kind: pipeline
type: docker
name: defaults
steps:
  - name:  {{ upper .input.stepName }}
    image: {{ .input.image }}
    commands:
      - {{ .input.commands }}