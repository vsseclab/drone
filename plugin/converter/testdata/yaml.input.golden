kind: pipeline
type: docker
name: defaults
steps:
  - name:  MY_STEP
    image: my_image
    commands:
      - my_command